enumerable.rb

```diff
@@ -0,0 +1,178 @@
+# rubocop:disable all
+module Enumerable
+  def my_each()
```

Comment:
- [ ]  [OPTIONAL] `def my_each()` This is better without parentheses. `def my_each` :star:  
https://rubystyle.guide/#method-parens